includes/class-evf-emails.php

```diff
@@ -45,6 +45,13 @@ class EVF_Emails {
 	 */
 	private $cc = false;
 
+		/**
```

Comment:
@ujw0l DocBlock position is wrong. Please fix this.

Reply:
Fixed 👍